modules/apps/frontend-taglib/frontend-taglib-clay-sample-web/src/main/resources/META-INF/resources/partials/cards.jsp

```diff
@@ -275,7 +275,6 @@ String SVG_FILE_TITLE = "lexicon.icon.camera.svg";
 	<div class="col-md-6" id="image-card-block">
 		<clay:user-card
 			actionDropdownItems="<%= cardsDisplayContext.getActionDropdownItems() %>"
```

Comment:
@dantewang @yuhai 
I am not going to merge this commit.

Reason is there is still usage of the method, you are removing usage and deprecation at the same commit. Let's delay this to second round, remove usage first, then remove deprecation.

Thanks.
Tina.